pkg/jx/cmd/shell.go

```diff
@@ -130,27 +131,23 @@ func (o *ShellOptions) Run() error {
 	newConfig := *config
 	newConfig.CurrentContext = ctxName
 
-	//clean old folders
```

Comment:
this was a bad thing to do even on linux as it mean if you ran `jx shell` inside a shell already started with `jx shell` then when you exitted the second subshell the kubeconfig was gone from the original `jx shell`